### PR TITLE
add InfofDepth (and other log level)

### PR DESCRIPTION
**Pull request by zhanghongnian**
add InfofDepth (and other log level)

### CODE REVIEW

File: glog.go [hunk 1]
```diff
@@ -729,6 +729,19 @@ func (l *loggingT) printf(s severity, format string, args ...interface{}) {
 	l.output(s, buf, file, line, false)
 }
 
+func (l *loggingT) printfDepth(s severity, depth int,format string, args ...interface{}) {
```
Comment (yangyi-bupt) [inline glog.go]: , 后空一格，下同

**Comment by yangyi-bupt** [comment]
lgtm

File: glog.go (hunk 1 above)
Comment (zhanghongnian) [reply glog.go]: 已改

**Comment by lftk** [comment]
LGTM

**Pull request merged**